backend/webapi/Features/Parties/ProfileStatus.Model.cs

```diff
@@ -251,6 +264,7 @@ public class PrescriptionRefillEformsSection : ProfileSection
         {
             internal override string SectionName => "prescriptionRefillEforms";
             public override string[] KeyWords => ["pharmacists", "rx"];
+            public override string ErrorReason => "Incorrect credential type being used to request enrolment.";
```

Comment:
As the same value is being used in multiple places can we define in constants and re-use in multiple places.